helpers/deploymentConfig.ts

```diff
@@ -183,46 +215,27 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
           jumpMultiplierPerYear: convertToUnit("5.0", 18),
           kink_: convertToUnit("0.8", 18),
           collateralFactor: convertToUnit("0.75", 18),
-          liquidationThreshold: convertToUnit("0.8", 18),
-          reserveFactor: convertToUnit("0.1", 18),
-          initialSupply: convertToUnit(9000, 18),
-          supplyCap: convertToUnit(5_500_000, 18),
-          borrowCap: convertToUnit(4_400_000, 18),
-          vTokenReceiver: vTreasuryAddress,
-        },
-        {
-          name: "Venus DOGE",
```

Comment:
Why are we removing vDOGE and vUSDT?

Reply:
Because they don't have the corresponding `tokenConfig` for these tokens. The alternative would be to specify both token configs and market configs for all the markets.